.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "parser": "typescript",
+  "singleQuote": false,
+  "semi": true,
+  "tabWidth": 2,
+  "useTabs": true
```

Comment:
not spaces? :)

Reply:
This one is from setup-ros, so keeping for consistency, but agree with you in the principle.